### ISSUE TITLE
Unable to resolve module "screens/login"

### ISSUE THREAD

**Issue by SFRohan** [facebook/react-native#19727]
I am very new to react-native and facing this issue for last 4 days.
```

Unable to resolve module "screens/login" from "Users/abc/Documents/MyApp/index.js"
Module name does not exist in the module map or in these directories /Users/abc/Documents/MyApp/node_modules/screens/,
/Users/abc/node_modules/screens

```
login is one of my screen under screens folder.

I am not getting why is it searching screens folder into node_modules folder.
This screens folder resides in root level of the project i.e. same level as of node_modules.

( This is an old app that I am working on ).

> Environment:
> react-native-cli: 2.0.1
> react-native: 0.43.3

Please somebody help !!!

**Comment by tijp** [comment]
If you import it like this it should work:
`import Login from './screens/login'`

or like this if you do not export the component as default:
`import { Login } from './screens/login'`

<sub>I think this question could be better asked on [StackOverflow](https://stackoverflow.com/questions/tagged/react-native)</sub>

**Comment by SFRohan** [comment]
Hi tijp,

THank you for your comment.
The project was running smooht earlier.

But then, I had to add library safari-view-controller & link it.

After I added library and ran react-native run-ios, it gave me an issue saying "not a npm folder. Run npm install & make sure you are inside react native folder " (kind of like this) ...

After I ran 'npm install', it started giving me this error.
I guess, its not updating packages correctly or something weird is happening.

I also tried yarn instead of npm install once. But the same thing happened.

Thanks,
Rohan.
